### DISCUSSION TITLE
Improved cryogenic and cryoplant model

### DISCUSSION THREAD

**Discussion by jonmaddock** [ukaea/PROCESS#2925] [Ideas]
By @mkovari on Nov 22, 2021, 16:18

The model for the cryogenic cooling power, and the electric power to provide this, is very old, and has a number of deficiencies.

The model is based on D.S. Slack, J.A. Kern, J.R., Miller, Cryogenic system design for a compact tokamak reactor, UCRL-98733, DE89 003176 (1989).
1.  Heat conduction through gravity support is based on these assumptions:  

| Average thermal conductivity of stainless steel between 300 K and 4.5 K | 10 W/(mK) |
|-------------------------------------------------------------------------|-----------|
| Stress in gravity support                                               | 67 MPa    |
| Length of gravity support                                               | 1 m       |

 
2.  Black body radiation load  
I don't know what this calculation is based on.  In any case, it seems to take account solely the inward-facing (plasma-facing) area of the TF coil legs, neglecting the outward facing surface.  

3.  AC losses.  This formula doesn’t seem to have anything to do with AC losses in the coils.  

4. The code does not take account of high-temperature superconducting current leads, which will certainly be used.  

5.  The PF coils seem to have been omitted, both for radiation and for current leads.

6.  We need to allow the TF and PF coils to be at different temperatures.  (See also #1462) 

7.  The hard-coded figure of 13% of the ideal Carnot efficiency is now being changed, but we need to make this relative efficiency as input, as in #1431. 

8.  See also #1253 (Superconducting busbars), #1656 and other cryogenic issues.
 
Comments @cjwgriesel  @jmorris-uk  @jonmaddock @ajpearcey ?

**Comment by mkovari** [comment]
@clmould @jonmaddock

**Reply by cjwgriesel** [reply]
what about the performance and efficiencies of the heat exchangers. Pressure drops in the superconducting circuits?

**Comment by mkovari** [comment]
![image](https://github.com/user-attachments/assets/177f2863-6686-47be-a0d1-a8e7fa1e98bb)
[Electrical Loads and Power Systems for the DEMO Nuclear Fusion Project (2020)](https://www.mdpi.com/1996-1073/13/9/2269)

[EFDA_D_2RBD4J - Technical Report 2023 for WPDES Task DES-NFS.FLUID.CRYO-T001](https://idm.euro-fusion.org/?uid=2RBD4J)
Includes a brief description of how the cryogenic heat loads are calculated, with results and several references.

See also this folder: [Cryo](https://idm.euro-fusion.org/?uid=2PJ7DN)
